_posts/2024-06-01-Announcing-Inbound-IPv6-support.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Announcing inbound IPv6 support in public preview"
+author_name: "Mads Damgård"
+toc: true
+toc_sticky: true
+---
+
+I am happy to announce the first part of our IPv6 implementation in App Service. Public preview of inbound IPv6 support for multi-tenant apps on Premium SKUs, Functions Consumption, Functions Elastic Premium, and Logic Apps Standard). We'll be adding IPv6 support in four stages.
+
+1. IPv6 inbound support (multi-tenant)
+1. IPv6 non-vnet outbound support (multi-tenant)
+1. IPv6 vnet outbound support (multi-tenant and App Service Environment v3)
```

Comment:
Has the ASE team confirmed any level of support? We are so far not claiming anything works on ASE

Reply:
I am also the ASE team :-) and driving the dialog with swift on supporting IPv6. Message is that they should have swift v2 ready by Oct and we can start implementing then - great risk of delays and no idea how long our implementation will take so a careful guess is mid-CY25. The list is also to say that it is not currently supported, but will be coming. I'll see if I can make it more clear.